google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -346,7 +346,7 @@ def expires_in
         #   to unset.
         #
         def expires_in= ttl
-          new_expiration_policy = Google::Pubsub::V1::ExpirationPolicy.new ttl: Convert.number_to_duration(ttl)
+          new_expiration_policy = Google::Cloud::Pubsub::V1::ExpirationPolicy.new ttl: Convert.number_to_duration(ttl)
```

Comment:
s/Pubsub/PubSub/

(`Google::Cloud::Pubsub` is an alias of `Google::Cloud::PubSub` right now, so this isn't strictly necessary, but better to clean it up.)